_data/tagging-status.yml

```diff
@@ -4203,11 +4217,10 @@
 
  - name: xltabular
    type: package
-   status: unknown
-   issues:
-   tests: false
-   tasks: needs tests
-   updated: 2024-07-15
+   status: currently-incompatible
+   issues: [170]
```

Comment:
even though it doesn't work, we should probably already list that it needs phase-III,table eventually

Reply:
Fixed